stories/core_components/GeoPushpinChart.tsx

```diff
@@ -293,4 +297,76 @@ storiesOf("Core components/GeoPushpinChart", module)
                 config,
             }),
         );
-    });
+    })
+    .add("with custom colors", () =>
+        screenshotWrap(
+            renderGeoPushpinChart({
+                projectId: "storybook",
+                location: ATTRIBUTE_LOCATION_GEOCHART,
+                size: MEASURE_SIZE_GEOCHART,
+                segmentBy: ATTRIBUTE_SEGMENT_GEOCHART,
+                config: {
+                    ...DEFAULT_CONFIG,
+                    colors: [
+                        "rgb(162, 37, 34)",
+                        "rgb(194, 153, 121)",
+                        "rgb(168, 194, 86)",
+                        "rgb(243, 217, 177)",
+                        "rgb(195, 49, 73)",
+                    ],
+                },
+            }),
+        ),
+    )
+    .add("with custom colors, color mapping for SegmentBy", () =>
+        screenshotWrap(
+            renderGeoPushpinChart({
+                projectId: "storybook",
+                location: ATTRIBUTE_LOCATION_GEOCHART,
+                size: MEASURE_SIZE_GEOCHART,
+                color: MEASURE_COLOR_GEOCHART,
+                segmentBy: ATTRIBUTE_SEGMENT_GEOCHART,
+                config: {
+                    ...DEFAULT_CONFIG,
+                    tooltipText: ATTRIBUTE_TOOLTIP_GEOCHART,
+                    ...CUSTOM_COLOR_PALETTE_CONFIG,
+                    colorMapping: [
+                        {
+                            predicate: uriMatch("/gdc/md/storybook/obj/23/elements?id=3"),
+                            color: {
+                                type: "guid",
+                                value: "03",
+                            },
+                        },
+                        {
+                            predicate: attributeItemNameMatch("Toy Store"),
+                            color: {
+                                type: "rgb",
+                                value: { r: 168, g: 194, b: 86 },
+                            },
+                        },
+                    ],
+                },
+            }),
+        ),
+    )
+    .add("with color mapping for Location", () =>
+        screenshotWrap(
+            renderGeoPushpinChart({
+                projectId: "storybook",
+                location: ATTRIBUTE_LOCATION_GEOCHART,
+                config: {
+                    ...DEFAULT_CONFIG,
+                    colorMapping: [
+                        {
+                            predicate: identifierMatch("30.df"),
+                            color: {
+                                type: "guid",
+                                value: "5",
+                            },
+                        },
+                    ],
+                },
+            }),
+        ),
+    );
```

Comment:
I see we have 3 new storybook cases.
Why are there other 4 images updated with new color?

Reply:
I modified data at [here](https://github.com/gooddata/gooddata-react-components/blob/bb742ee496f27b82eace5f2e53412a5f383c072e/stories/data/geoChart/fixtures.js#L397) so the category legend have 4 items now (eg: General Goods, Toy Store, Convenience, Speciality)